app.py

```diff
@@ -34,7 +34,7 @@ def main():
     """
     if current_user.is_authenticated:  # avoid "must login" message
         return redirect("index")
-    return redirect("login")
+    return render("login")
```

Comment:
`"login"` should be removed here; the `render` helper function does not need to know the path (`index.html` is always rendered). Also, "login" is not being rendered in this case either way; the landing page is. `render()` should be fine.

Reply:
That worked thanks sorry for not understanding your original instructions.